src/D2L.CodeStyle.Annotations/D2L.CodeStyle.Annotations.csproj

```diff
@@ -21,4 +21,8 @@
     <IncludeSymbols>True</IncludeSymbols>
   </PropertyGroup>
 
+  <ItemGroup>

```

Comment:
Is this needed? This seems unusual. I only seen this when adding an empty directory, but the folder looks populated